package.json

```diff
@@ -25,15 +27,14 @@
     "lint:js": "eslint . --cache --cache-location=.cache/eslint --ext .js,.jsx,.json",
     "lint:md": "remark .",
     "publish": "lerna publish",
+    "start": "serve ./docs/out --single",
```

Comment:
add `serve` as dev dependency? i think you are mixing in some of your docsv2 stuff in here?